### PR TITLE
Add Bintray integration for release publishing

### PR DESCRIPTION
**Pull request by sav007**

### CODE REVIEW

File: apollo-android-support/build.gradle [hunk 1]
```diff
@@ -48,3 +48,4 @@ dependencies {
 }
 
 apply from: rootProject.file('gradle/gradle-mvn-push.gradle')
```
Comment (digitalbuddha) [inline apollo-android-support/build.gradle]: Do we need both?

File: apollo-rx2-support/gradle.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
 POM_ARTIFACT_ID=apollo-rx2-support
-POM_NAME=Apollo GraphQL Rx2Support
+POM_NAME=Apollo GraphQL Rx2 Support
 POM_DESCRIPTION=Apollo GraphQL RxJava2 bindings
 POM_PACKAGING=jar
```
Comment (digitalbuddha) [inline apollo-rx2-support/gradle.properties]: Missing new line eof

File: build.gradle [hunk 3]
```diff
@@ -1,6 +1,16 @@
 subprojects {
   apply from: rootProject.file("gradle/dependencies.gradle")
 
+  File localProperties = project.rootProject.file('local.properties')
```
Comment (digitalbuddha) [inline build.gradle]: Can this block be in the bintray gradle file or in its own. Just to keep things cleaner here.

File: gradle/bintray.gradle [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    websiteUrl = POM_URL
+    vcsUrl = POM_SCM_URL
+    licenses = [POM_LICENCE_NAME]
+    publish = true
+    publicDownloadNumbers = true
+    version {
+      desc = POM_DESCRIPTION
+    }
+  }
+}
```
Comment (digitalbuddha) [inline gradle/bintray.gradle]: Newline

File: apollo-android-support/build.gradle (hunk 1 above)
Comment (sav007) [reply apollo-android-support/build.gradle]: Yes, snapshots still publishing to sonatype

**Comment by sav007** [comment]
No we don't need to change the travis as it publishes the snapshot versions that are still uses sonatype

**Pull request merged**